layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutViewModel.kt

```diff
@@ -17,4 +17,9 @@ internal class FoldableLayoutViewModel : ViewModel() {
      * Application screen state.
      */
     var windowLayoutInfo: WindowLayoutInfo? = null
+
+    /**
+     * Current SurfaceDuoLayout configuration
```

Comment:
Here also, please change it to "FoldableLayout":)

Reply:
Done